pkg/resource/prunercronjob.go

```diff
@@ -0,0 +1,226 @@
+package resource
+
+import (
+	"fmt"
+	"os"
+
+	batchv1 "k8s.io/api/batch/v1"
+	batchapi "k8s.io/api/batch/v1beta1"
+	kcorev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	batchset "k8s.io/client-go/kubernetes/typed/batch/v1beta1"
+	batchlisters "k8s.io/client-go/listers/batch/v1beta1"
+
+	imageregistryapiv1 "github.com/openshift/api/imageregistry/v1"
+	imageregistryv1listers "github.com/openshift/client-go/imageregistry/listers/imageregistry/v1"
+	"github.com/openshift/cluster-image-registry-operator/defaults"
+	"github.com/openshift/cluster-image-registry-operator/pkg/parameters"
+)
+
+var (
+	defaultSuspend                          = false
+	defaultSchedule                         = "*/1 * * * *"
+	defaultStartingDeadlineSeconds    int64 = 60
+	defaultFailedJobsHistoryLimit     int32 = 3
+	defaultSuccessfulJobsHistoryLimit int32 = 3
+	defaultKeepTagRevisions                 = 3
+	defaultKeepYoungerThan                  = "60m"
```

Comment:
nit - `const` for things that are primitives? Or does this break the requirement for pointers?

Reply:
Correct, I was unable to use a constant here.